### ISSUE TITLE
bug: Pagination accessibility issue

### ISSUE THREAD

**Issue by scott-buchanan** [cds-snc/gcds-components#891]
### Prerequisites | Prérequis

- [x] I have read the [Contributing Guidelines](https://github.com/cds-snc/gcds-components/blob/main/CONTRIBUTING.md). | J’ai lu les [Lignes directrices sur les contributions](https://github.com/cds-snc/gcds-components/blob/main/CONTRIBUTING.md).
- [x] I agree to follow the [Code of Conduct](https://github.com/cds-snc/gcds-components/blob/main/CODE_OF_CONDUCT.md). | J’accepte de me conformer au [Code de conduite](https://github.com/cds-snc/gcds-components/blob/main/CODE_OF_CONDUCT.md).
- [x] I have searched for [existing issues](https://github.com/cds-snc/gcds-components/issues) that already report this problem, without success. | J’ai vérifié parmi [les tickets (issues) existants](https://github.com/cds-snc/gcds-components/issues) que ce sujet n’avait pas déjà été soumis.

### GC Design System Components Package and Version | Paquet et version des composants de Système de design GC

@cdssnc/gcds-components-vue@0.36.0

### Current Behavior | Comportement observé

Integrating components into vue, Storybook accessibility tests revealed this.

![Image](https://github.com/user-attachments/assets/7245a7d2-55e0-4126-bd37-f9b03a2c3c22)

### Expected Behavior | Comportement attendu

No accessibility violation.

### System Info | Information sur le système

```shell

```

### Steps to Reproduce | Étapes pour reproduire le bogue

Place component on page.
